dotcom-rendering/src/layouts/CrosswordLayout.tsx

```diff
@@ -309,16 +310,22 @@ export const CrosswordLayout = (props: Props) => {
 							</GridItem>
 							<GridItem area="right-column">
 								<RightColumn showFrom="wide">
-									{renderAds ? (
-										<AdSlot
-											position="right"
-											display={format.display}
-											isPaidContent={isPaidContent}
-											shouldHideReaderRevenue={
-												article.shouldHideReaderRevenue
-											}
-										/>
-									) : null}
+									<div
+										css={css`
+											margin-top: ${remSpace[3]};
+										`}
+									>
+										{renderAds ? (
```

Comment:
I wonder if `renderAds` is false, we should adjust the grid to remove the right column entirely and let the crossword body take up the remaining space? One for a follow up though.